### ISSUE TITLE
Add reverse hostname to online_failover_ip

### ISSUE THREAD

**Issue by meyskens** [src-d/terraform-provider-online-net#34]
We should add the ability to change the reverse hostname of a failover ip in the `online_failover_ip` resource instead of a separate one

**Comment by nclandrei** [comment]
Hi @meyskens! Can I work on this one? 🙂

**Comment by meyskens** [comment]
@nclandrei sure! If you need any help feel free to ping me for questions

**Comment by meyskens** [comment]
@nclandrei still working on that one?

**Comment by nclandrei** [comment]
Hi @meyskens! I really do apologize for being so unresponsive but I had some urgent personal matters and had to fly back home for a while. Is it ok if I finish this today or tomorrow max? 🙂 Again, sorry for not communicating that I left.

**Comment by nclandrei** [comment]
@meyskens - so if I understand this correctly, the changes should be something similar to the ones described in #10 (the implementation for this task should be similar as what was done for the reverse hostname for the server)? I had a look through the code to understand it properly and was just wondering if I’m on the right track 😁

**Comment by meyskens** [comment]
That's correct, in #10 the idea was to make a `reverse_hostname` resource while here the idea is to have like

```hcl
resource "online_failover_ip" "test" {
  ip = "127.0.0.1"
  hostname = "localhost.sourced.tech"
}
```
